controllers/topics.controllers.js

```diff
@@ -0,0 +1,11 @@
+const { 
+    selectAllTopics 
+} = require("../modules/topics.modules")
```

Comment:
A quick note on a typo: a `module` is a word use to refer generically to files within Nodejs. The word you're looking for is `models` - for the sake of clarity for other devs reading your code, I would recommend renaming this folder & file!

Reply:
oooh, yeah, good catch. Thank you